src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -339,6 +340,18 @@ web-ext run --start-url www.mozilla.com --start-url developer.mozilla.org
 
 Environment variable: `$WEB_EXT_START_URL`
 
+#### `--watch-ignored`
+A list of paths and globs patterns that should not be watched for changes. This is useful if you want to explicitly prevent web-ext from watching part of the extension directory tree, e.g. the node_modules folder.
+
+```shell
+web-ext run --watch-ignored path/to/file.js path/to/directory/to-be/ignored
+```
```

Comment:
it may be good to explicitly include a glob pattern in the example snippet.
(as a side note, watchpack does internally use [`glob-to-regexp`](https://www.npmjs.com/package/glob-to-regexp) for the `ignore` parameter, [see here](https://github.com/webpack/watchpack/blob/055e008168d89d63c273f97ae010303b533c936c/lib/watchpack.js#L28-L29), with the `extended` and `globStar` options set to `true`, and so you can find some more details about the glob patterns formats supported in the [glob-to-regexp readme here](https://www.npmjs.com/package/glob-to-regexp)).